src/community/backup-restore/core/pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.geoserver.community</groupId>
+		<artifactId>gs-backup-restore</artifactId>
+		<version>2.10-SNAPSHOT</version>
+	</parent>
+
+	<groupId>org.geoserver.community.backuprestore</groupId>
+	<artifactId>gs-backup-restore-core</artifactId>
+	<packaging>jar</packaging>
+	<name>GeoServer Config Backup And Restore Core Module</name>
+
+	<properties>
+		<spring.batch.version>3.0.7.RELEASE</spring.batch.version>
+	</properties>
+
+	<dependencies>
+
+		<!-- Spring dependencies -->
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-core</artifactId>
+			<version>${spring.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-messaging</artifactId>
```

Comment:
I remember I had to put the dep in order to add some utility classes at the beginning. I will check if it still needed or not. Did not check after refactoring.